frontend/assets/stylesheets/_extends-only.scss

```diff
@@ -28,7 +28,7 @@
     position: absolute;
     top: rem($gs-gutter / 2);
     right: rem($gs-gutter / 2);
-    background-color: $border-bottom-color;
+    background-color: rgba(255, 255, 255, .29);
```

Comment:
This is deliberate. Our variables are getting a bit messy, particularly with border colours, so I opted to hard-code this colour so I could clean up the border variables a little bit.